### ISSUE TITLE
Create helper script for appending asterisks to students names

### ISSUE THREAD

**Issue by namgo** [neuropoly/bibeasy#19]
Following up on https://github.com/neuropoly/bibeasy/issues/10 we decided that a quick script to add asterisks to students names was a more feasible way to meet an upcoming deadline, and that issue 10 would continue to be worked on over the next while.

@jcohenadad At this point I have what I believe to be a solution to adding asterisks safely, alongside a better understanding of the underlying schema. @SomeoneInParticular and @mguaypaq both assisted in this process and I am very grateful. I'm about to submit a draft PR which will contain:

- the script to add asterisks to student names matching `bibeasy.formatting.STUDENTS`
- some additional matches for the `STUDENTS` list (more names without accented characters)
- possibly my de-naming script for testing if I can get it working for CCV

@jcohenadad should I include the updated xml you have provided or leave that out?

I'm currently struggling to test the generated xml, I'm removing as much information as I can from the source XML about who it is from, but the website continues to give me an error related to account mismatches.

Notes:
- kousu 's previous observation of unique identifiers for every entry may be what's getting in the way
  - This might not actually be the problem, as I was previously able to import Nick's xml without regenerating identifiers
  - Julien's xml is just much larger
  - I'm currently working to extract only the relevant sections for testing
- ccv-cvc.ca doesn't care if the xml is pretty-printed or not
- it also doesn't care if `\r\n` is used, so `\n` is fine

The latter note is somewhat important for python's xml parser.

**Comment by jcohenadad** [comment]
> @jcohenadad should I include the updated xml you have provided or leave that out?

feel free to add it-- all that info is public anyway

**Comment by namgo** [comment]
I'm working through some tests today (I know it's a holiday but I really want to get this done). I'm struggling to properly de-identify Julien's CCV, so I'm exporting my own and testing with that.

I've added `--append` to the script to assist testing. I had previously obtained Nick's testing xml, there was less to remove there and I had gotten that imported a month or so ago.

The script appears to work!!

with `python3  student_asterisk_helper.py -i ~/Downloads/CCV-10259659\(3\).xml -o /tmp/ccv-nathan.xml -a 'Cohen-Adad J'`

The exported preview CV goes from 

[CCV-NathanGorvett-2.pdf](https://github.com/user-attachments/files/17369006/CCV-NathanGorvett-2.pdf)

To

[CCV-NathanGorvett-3.pdf](https://github.com/user-attachments/files/17369005/CCV-NathanGorvett-3.pdf)

It appears to be completely valid, the asterisk is kept for Julien (who I've added as an append, to be clear).

Then I ran it over once more without appending Julien and the asterisk was properly removed (as it's not in the list):

`python3  student_asterisk_helper.py -i ~/Downloads/CCV-10259659\(3\).xml -o /tmp/ccv-nathan.2.xml`

[CCV-NathanGorvett-4.pdf](https://github.com/user-attachments/files/17369026/CCV-NathanGorvett-4.pdf)
